<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ClosingEntry</class>
 <widget class="QWidget" name="ClosingEntry">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>540</width>
    <height>429</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>10</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Daily Closing </string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <layout class="QFormLayout" name="formLayout">
       <item row="1" column="0">
        <widget class="QLabel" name="noOfShiftsLabel">
         <property name="text">
          <string>No. Of Shifts</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="noOfShiftsLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="billsFromLabel">
         <property name="text">
          <string>Bills From</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="billsFromLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="billsToLabel">
         <property name="text">
          <string>Bills To</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="billsToLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="salesTotalLabel">
         <property name="text">
          <string>Sales Total</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="salesTotalLineEdit"/>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="purchaseTotalLabel">
         <property name="text">
          <string>Purchase Total</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLineEdit" name="purchaseTotalLineEdit"/>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="cardSalesLabel">
         <property name="text">
          <string>Card Sales</string>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLineEdit" name="cardSalesLineEdit"/>
       </item>
       <item row="8" column="0">
        <widget class="QLabel" name="creditSalesLabel">
         <property name="text">
          <string>Credit Sales</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLineEdit" name="creditSalesLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QFormLayout" name="formLayout_3">
       <item row="0" column="0">
        <widget class="QLabel" name="openingCashLabel">
         <property name="text">
          <string>Opening Cash</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="openingCashLineEdit"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="cashSalesLabel">
         <property name="text">
          <string>Cash Sales</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="cashSalesLineEdit"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="cashPurchaseLabel">
         <property name="text">
          <string>Cash Purchase</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="cashPurchaseLineEdit"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="cashReceiptsLabel">
         <property name="text">
          <string>Cash Receipts</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLineEdit" name="cashReceiptsLineEdit"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="cashPaymentsLabel">
         <property name="text">
          <string>Cash Payments</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="cashPaymentsLineEdit"/>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="closingCashLabel">
         <property name="text">
          <string>ClosingCash</string>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="closingCashLineEdit"/>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPlainTextEdit" name="narration">
     <property name="placeholderText">
      <string>Remarks</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="ShiftsGroup">
     <property name="title">
      <string>Shifts</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QTableWidget" name="tableWidget"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="OkButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="CancelButton">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="text">
        <string>Print</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
